sdks/python/apache_beam/transforms/ptransform.py

```diff
@@ -328,7 +331,7 @@ def visit_dict(self, pvalueish, sibling, pairs, context):
         self.visit(p, sibling, pairs, context)
 
 
-class PTransform(WithTypeHints, HasDisplayData):
+class PTransform(WithTypeHints, HasDisplayData, Generic[InputT, OutputT]):
```

Comment:
Adding `Generic[InputT, OutputT]` breaks this test: `<apache_beam.transforms.periodicsequence_test.PeriodicSequenceTest testMethod=test_periodicimpulse_default_start>`. 

```
    def test_periodicimpulse_default_start(self):
      default_parameters = inspect.signature(PeriodicImpulse).parameters
>     it = default_parameters["start_timestamp"].default
E     KeyError: 'start_timestamp'
```

Do you know why this happens?

Reply:
Fixed.